### DISCUSSION TITLE
Graphs

### DISCUSSION THREAD

**Discussion by aalibes** [creativetimofficial/soft-ui-dashboard#20] [Q&A]
Hi,

Do you have any example we can follow of a graph using data taken from a mysql db?

Thanks

**Comment by rarestoma** [comment]
Hi @aalibes,

Thank you for working with our products.

Unfortunately, we do not have any examples, but I found this article that might help: https://dyclassroom.com/chartjs/chartjs-how-to-draw-bar-graph-using-data-from-mysql-table-and-php.

Regards,
Rares

**Reply by aalibes** [reply]
Thanks!
